release-pipeline/Jenkinsfile-release-pipeline.groovy

```
@@ -14,6 +14,8 @@ properties([
                 booleanParam(name: 'skipTests', defaultValue: true, description: 'Check to skip the Run Tests Stage'),
                 string(name: 'mvn_test_args_param', defaultValue: 'clean verify -Djarsigner.skip', description: ''),
                 booleanParam(name: 'tag_release_param', defaultValue: true, description: '<hr>'),
+                string(name: 'exchange_env_param_list', defaultValue: 'qax,stgx', description: '''<p>Comma separated Exchange environments where the Extension is going to be deployed.</p>
```

Lets confirm with @fsgonz but as in 3.x we don't run tests in FD we can use all the environments in the default value:  
```suggestion
                string(name: 'exchange_env_param_list', defaultValue: 'qax,stgx,prod,produ-eu', description: '''<p>Comma separated Exchange environments where the Extension is going to be deployed.</p>
```

---

Yes, we can set this as default. Usually we directly deploy to all environments.